R/reporter-junit.R

```diff
@@ -0,0 +1,153 @@
+#' @include reporter.R
+NULL
+
+classnameOK <- function(text) {
+  gsub("[ \\.]", "_", text)
+}
+
+add_broken <- function (parent, type, message) {
```

Comment:
I followed the previous author here but type should be either the "exception class" (for error) or "type of the assertion" for failure). However, it doesn't seem we have an equivalent value here and XSD requires a type. Do you have a better idea what to put there?